coglib/fmri/readme.md

```diff
@@ -142,6 +149,7 @@ Preprocess (f)MRI data. Perform visual inspection of each runs data (see `./bids
 	fmriprep_sub.py ./bids -w ./scratch/fmriprep_workdir --time 80 --mem_mb 30000 -n 6 -a " --ignore sbref slicetiming --output-spaces T1w MNI152NLin2009cAsym"
 ```
 
+Run time ~= 2 days per subject
```

Comment:
prep is at most 2 days, when the HPC had some hiccups. more realistic is probably 1 day